### PR TITLE
Update Dc11SchemaPublicationFormatAdapter.php

### PR DESCRIPTION
**Pull request by WillemStruiksma**
Missing DOI output in OAI-export OMP.

See: https://forum.pkp.sfu.ca/t/missing-doi-in-oai-export-omp-3-4-0-5/89063/2

Regards,
@ugp2

### CODE REVIEW

**Comment by asmecher** [comment]
@ewhanson, would you mind reviewing this? (See linked forum post.)

File: plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.php [hunk 1]
```diff
@@ -179,7 +179,13 @@ public function extractMetadataFromDataObject(&$publicationFormat)
...
             }
         }
-
+        if ($context->areDoisEnabled()) {
+            $doi = $publication->getData('doiObject');
+            if ($doi)
+            {
+                $dc11Description->addStatement('dc:relation', $doi->getData('doi'));
+            }
+        }
```
Comment (ewhanson) [inline plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.php:188]: I would preserve the blank line between the public identifiers section and the identifier: others section just to make it more readable.

**Review by ewhanson — CHANGES_REQUESTED** [review]
Looks good! 👍 Just one minor style request.

File: plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.php (hunk 1 above)
Comment (WillemStruiksma) [reply plugins/metadata/dc11/filter/Dc11SchemaPublicationFormatAdapter.php:188]: Ok, thanx, I'll change that.

**Review by ewhanson — APPROVED** [review]

**Comment by ewhanson** [comment]
Thanks @WillemStruiksma. Merged!

**Pull request merged**